README.rst

```diff
@@ -157,12 +157,15 @@ Contributing to Jet
 We welcome new contributions - simply fork the Jet repository and make a
 `pull request <https://help.github.com/articles/about-pull-requests/>`_
 containing your contribution.  All contributers to Jet will be listed as authors
-on the releases.  See our `changelog <.github/CHANGELOG.md>`_ for more details.
+on the releases.  See our `changelog
+<https://raw.githubusercontent.com/XanaduAI/jet/.github/CHANGELOG.md>`_ for more
+details.
 
 We also encourage bug reports, suggestions for new features and enhancements,
 and even links to cool projects or applications built using Jet.  Visit the
```

Comment:
Do you follow the 2 spaces convention, or is this a typo?
```suggestion
and even links to cool projects or applications built using Jet. Visit the
```

Reply:
Sorry, this is just an old habit of mine. Fixed!